.eslintrc

```diff
@@ -40,7 +43,7 @@
     "consistent-return": 0,
     "no-param-reassign": 0,
     "comma-dangle": 2,
-    "no-console": 1,
+    "no-console": 0,
```

Comment:
This should be set to 1 so as to remind us to remove console statements before pushing.

Reply:
Alright. I made it zero when my server was failing because of console.log. I will change it now. Thanks for the observation.

Reply:
> This should be set to 1 so as to remind us to remove console statements before pushing.

Done